meteor-app/imports/api/parts/server/publications.js

```diff
@@ -0,0 +1,6 @@
+import { Meteor } from 'meteor/meteor';
+import Parts from '../parts';
+
+Meteor.publish('all.parts', () => {
```

Comment:
This isn't a problem yet, but it's not great practice to simply publish all records in a collection, because it pushes your entire collection up to the client, and there are the following problems with that:
1) An attacker can scrape your entire parts database, including retail and wholesale prices
2) Performance could be a problem, as you are pushing all of the data out to the client, on a slow connection this makes a different to load times, and also on mobile devices the memory may be limited. Also the server keeps a copy of the published data in memory for each client, so when you scale up to many clients even the server isn't efficient